src/modules/tools/endstops/Endstops.cpp

```diff
@@ -146,6 +147,10 @@ void Endstops::on_module_loaded()
 // Get config
 void Endstops::on_config_reload(void *argument)
 {
+    if (THEKERNEL->robot->actuators.size() > 3) {
+        THEKERNEL->streams->printf("Please implement trim for >3 actuators");
```

Comment:
Right now I'd leave endstops unchanged from what it was (other than using the new typedef). Handling multiple endstops can be left for when that use case actually evolves.
left as it was it would continue to work regardless of how many motors were defined.